content/english/facts/soy-is-bad-for-you.md

```diff
@@ -0,0 +1,191 @@
+> Soy's phytoestrogens cause a variety of negative health effects including reducing male fertility.
+
+See our article: [Soy contains estrogen]({{<ref "/facts/soy-contains-estrogen">}}) for more information on phytoestrogens.
+
+# Reality
+
+> 1. Soy contains phytoestrogens which are structurally similar to human estrogen (specifically *isoflavones*) which can have hormonal effects on the body.
+> 2. The maximum safe dosages are not well understood. 
+> 3. It is _unlikely_ that you will ingest a high enough dosage to affect your health as part of a balanced diet provided you do not have any special circumstances (e.g. hypothyroidism, infancy, etc.). But you should ensure that you vary your protein sources and consult your dietitian for special circumstances (e.g. hypothyroidism, infancy, etc.) see *Soy, Soy Foods and Their Role in Vegetarian Diets - 2018* below.  
+> 4. All articles on the topic caveat their conclusions with: 'more research is needed'.  
+>  
+> \- TalkVeganToMe's layman's understanding from surveying the material available online
+
+This is a highly complex and fast moving topic in terms of research. We've discounted a number of the most popular pro-soy sources (see *Conflicts of Interest* below). We've also intentionally put the position of the Vegan Society to one side until we understand the original sources properly (see *Vegan Society* below).
+
+What remains is the truest representation of the research we've surveyed from our layman's ability to digest them. Despite assessing nearly [80 articles and metaanalyses](https://docs.google.com/spreadsheets/d/e/2PACX-1vTayf4Bn7PT7xHNrQ_ehIQV_8gTWRLN9of5Kfm_GuPf9CAPtJ5hZfxEAvTvZub_orWOuPrCxIr7RpTq/pubhtml) we have relied particularly heavily on [Soy, Soy Foods and Their Role in Vegetarian Diets - Nutrients - 2018](https://web.archive.org/web/20191002195620/https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5793271/pdf/nutrients-10-00043.pdf/) as it is a highly relevant, highly recent, and seemingly unbiased metaanalaysis of the research.
+
+You may come to a different conclusion based on the evidence provided, which is why we have cited, and quoted, and linked to our sources so you can make your own mind up. If you wish to reach us regarding anything we've said in this article, please contact [support@talkveganto.me](mailto:support@talkveganto.me)
+
+
+## 1. Phytoestrogens can have hormonal (and anti-hormonal) effects.
+
+This was our biggest surprise in researching this topic, as we had heard over and over again that soy contained _plant_ estrogen, you don't need to worry about plant estrogen as they can't interact with the human body hormonally!
+
+Unfortunately phytoestrogens are not so easily dismissed. See *[Soy contains estrogen]({{<ref "/facts/soy-contains-estrogen">}})* for more.
+
+## 2. Dosage is hard to understand
+
+This assertion is based on:
+
+1. How much phytoestrogen is in food is difficult to measure and varies wildly
+2. How much is safe to eat is not well understood
```

Comment:
While you clarify below; pint 1 makes it sound like taking a specific measurement is difficult... whereas its just other factor result in phytoestrogen counts that vary wildly. Maybe just drop the difficult to measure part?

Reply:
Good shout. I'll change that.